app/util/crowd/populate_db.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+###################    Variables section         ###################
+# Command to install psql client for Amazon Linux 2.
+# In case of different distributive, please adjust accordingly or install manually.
+INSTALL_PSQL_CMD="amazon-linux-extras install -y postgresql10"
+
+# DB config file location (dbconfig.xml)
+DB_CONFIG="/usr/lib/systemd/system/crowd.service"
+
+# Depending on Crowd installation directory
+CROWD_VERSION_FILE="/media/atl/crowd/shared/crowd.version"
+
+# DB admin user name, password and DB name
+CROWD_DB_NAME="crowd"
+CROWD_DB_USER="postgres"
+CROWD_DB_PASS="Password1!"
+
+# Crowd version variables
+SUPPORTED_CROWD_VERSIONS=(4.3.0)
+
+if [[ ! $(systemctl status crowd) ]]; then
+  echo "The Crowd service was not found on this host." \
+  "Please make sure you are running this script on a host that is running Crowd."
+  exit 1
+fi
+
+CROWD_VERSION=$(sudo su crowd -c "cat ${CROWD_VERSION_FILE}")
+if [[ -z "$CROWD_VERSION" ]]; then
+  echo The $CROWD_VERSION_FILE file does not exists or emtpy. Please check if CROWD_VERSION_FILE variable \
+  has a valid file path of the Crowd version file or set your Cluster CROWD_VERSION explicitly.
+  exit 1
+fi
+echo "Crowd Version: ${CROWD_VERSION}"
+
+# Datasets AWS bucket and db dump name
+DATASETS_AWS_BUCKET="https://centaurus-datasets.s3.amazonaws.com/crowd"
+DATASETS_SIZE="large"
+DB_DUMP_NAME="db.dump"
+
+###################    End of variables section  ###################
+
+# Check if Crowd version is supported
+if [[ ! "${SUPPORTED_CROWD_VERSIONS[*]}" =~ ${CROWD_VERSION} ]]; then
+  echo "Crowd Version: ${CROWD_VERSION} is not officially supported by Data Center App Performance Toolkit."
+  echo "Supported Crowd Versions: ${SUPPORTED_CROWD_VERSIONS[*]}"
+  echo "!!! Warning !!! It is not revertable operation and may break your Crowd instance."
```

Comment:
let's change this warning to a message than dump from version ${SUPPORTED_CROWD_VERSIONS[0]} would be used.